packages/discord.js/src/util/Formatters.js

```diff
@@ -24,6 +25,16 @@ const {
  * Contains various Discord-specific functions for formatting messages.
  */
 class Formatters extends null {
+  /**
+   * Formats an application command name and id into an application command mention.
+   * @method applicationCommandMention
+   * @memberof Formatters
+   * @param {string} commandName The name of the application command.
+   * @param {string} commandId The id of the application command.
+   * @returns {string}
+   */
+  static applicationCommandMention = applicationCommandMention;
```

Comment:
could you add this to ApplicationCommand#toString()?

Reply:
And how would subcommands (and subcommand groups) be handled? Would they just be ignored?

Reply:
Well yeah there's no way to mention a subcommand

Reply:
like I said you can write any text so I think toString should only put the name of the command

Reply:
062bcccaf14d5c2a02693e3ece09dca1e8615af2